src/Phinx/Wrapper/TextWrapper.php

```diff
@@ -107,6 +110,45 @@ public function getRollback($env = null, $target = null)
         return $this->executeRun($command);
     }
 
+    /**
+     * Set the environment
+     *
+     * @param string $environment
+     */
+    public function setEnvironment($environment = null) {
```

Comment:
What's the reason for these? Nothing seems to use them, and they can already be modified by passing the argument to the specific method.